ddtrace/ddtrace.stub.php

```diff
@@ -0,0 +1,1051 @@
+         */
+        public function jsonSerialize(): mixed {}
+
+        /**
+         * Consumes distributed tracing headers, from which a span link will be constructed.
+         *
+         * @param array|callable(string):mixed $headersOrCallback Either an array with a lowercase header to value mapping,
+         * or a callback, which given a header name for distributed tracing, returns the value it should be updated to.
+         */
+        public static function fromHeaders(array|callable $headersOrCallback): SpanLink {}
+    }
+
+    class SpanData {
+        /**
+         * @var string|null The span name
+         */
+        public string|null $name = "";
+
+        /**
+         * @var string|null The resource you are tracing
+         */
+        public string|null $resource = "";
+
+        /**
+         * @var string|null The service you are tracing. Defaults to active service at the time of span creation (i.e.,
+         * the parent span), or datadog.service initialization settings if no parent exists
+         */
+        public string|null $service = "";
+
+        /**
+         * @var string The environment you are tracing. Defaults to active environment at the time of span creation
+         * (i.e., the parent span), or datadog.env initialization settings if no parent exists
+         */
+        public string $env = "";
+
+        /**
+         * @var string The version of the application you are tracing. Defaults to active version at the time of
+         * span creation (i.e., the parent span), or datadog.version initialization settings if no parent exists
+         */
+        public string $version = "";
+
+        /**
+         * @var string[] Meta struct can be used to send any data to the backend. The peculiarity of meta struct is
+         * that the values are encoded with msgpack when sent to the agent. The values are first encoded to msgpack
+         * and then, encoded again with msgpack to binary
+         */
+        public array $meta_struct = [];
+
+        /**
+         * @var string|null The type of request which can be set to: web, db, cache, or custom (Optional). Inherited
+         * from parent.
+         */
+        public string|null $type = "";
+
+        /**
+         * @var string[] $meta An array of key-value span metadata; keys and values must be strings.
+         */
+        public array $meta = [];
+
+        /**
+         * @var float[] $metrics An array of key-value span metrics; keys must be strings and values must be floats.
+         */
+        public array $metrics = [];
+
+        /**
+         * @var \Throwable|null $exception An exception generated during the execution of the original function, if any.
+         */
+        public \Throwable|null $exception = null;
+
+        /**
+         * @var string The unique identifier of the span
+         */
+        public readonly string $id;
+
+        /**
+         * @var SpanLink[] $spanLinks An array of span links
```

Comment:
The name in `@var` differs from actual property name. Could you please update the names to the correct one?